editor.planx.uk/src/@planx/components/Checklist/Public/components/ExclusiveChecklistItem.tsx

```diff
@@ -0,0 +1,33 @@
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import { FormikProps } from "formik";
+import React from "react";
+import FormWrapper from "ui/public/FormWrapper";
+import ChecklistItem from "ui/shared/ChecklistItem/ChecklistItem";
+
+import { Option } from "../../../shared";
+
+export const ExclusiveChecklistItem = ({
+  exclusiveOrOption,
+  changeCheckbox,
+  formik,
+}: {
+  exclusiveOrOption: Option;
+  changeCheckbox: (id: string) => () => void;
+  formik: FormikProps<{ checked: Array<string> }>;
+}) => (
+  <FormWrapper key={exclusiveOrOption.id}>
+    <Grid item xs={12} key={exclusiveOrOption.data.text}>
+      <Typography width={36} display="flex" justifyContent="center">
+        or
+      </Typography>
+
```

Comment:
nit: I personally do not like empty rows within `return` blocks! Same goes for this line below: https://github.com/theopensystemslab/planx-new/pull/4088/files#diff-b200eeb6b69eea54e2d875c01212d93f0b029698e062e35dcf88c337fedad06dR125